Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2001,6 +2008,7 @@
       Dead:
         Base: narsian_dead
   - type: MeleeWeapon
+    useWeaponAsAttackAnimation: false
```

Comment:
Setting this for every single mob is very tedious so please find an alternative.

Reply:
isnt there a base melee mob

Reply:
removed entirely